### ISSUE TITLE
NVDA insists on repeating the word "caption" as each line and paragraph gains focus with Firefox and Chrome

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#15505]
This is an issue which doesn't affect JAWS.
### Steps to reproduce:
1. Go to https://www.wired.com/gallery/best-language-learning-apps
2. Locate the 6th heading on the page which says: "Do Language Learning Apps Work?" It's a level-2 heading.
### Actual behavior:
As you press Down arrow to read the rest of the page line by line, or Control+Down to read paragraph by paragraph, or NVDA key+Down to read continuously, NVDA starts each line with the word, "caption," which is really annoying. So with each line gaining focus, we'll hear "caption" first.
### Expected behavior:
The word "caption" should be announced just once, or at least for each paragraph. This is also handled properly by JAWS which doesn't repeat the word "caption" at all.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Alpha 29323, 2023.3 Beta 1, and 2023.2.
#### Windows version:
Both 10 and 11.
#### Name and version of other software in use when reproducing the issue:
Both Chrome and Firefox 117
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - as mentioned above
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by ultrasound1372** [comment]
A possible desirable behavior would be to treat it like a container, similar to, say, block quotes, where it would say caption and out of caption. Knowing where the caption ends is a good thing but having it reported across each line is indeed annoying. Furthermore, if you are inside a caption and use k to navigate to a link that is inside that caption it still says caption when reading that link.

**Comment by amirsol81** [comment]
@ultrasound1372 This is a great solution IMO.

**Comment by XLTechie** [comment]
@ultrasound1372's suggestion seems best to me also. I too find the current 
behavior annoying.

**Comment by seanbudd** [comment]
See also #14349, #14874

**Comment by Adriani90** [comment]
Closing as duplicate of #14874.
